core/src/main/scala/cats/collections/HashMap.scala

```diff
@@ -617,7 +617,7 @@ object HashMap extends HashMapInstances with compat.HashMapCompatCompanion {
         this
       else if (contents.toVector.lengthCompare(2) == 0) {
         // There will no longer be any collisions once the key is removed
-        val keepIndex = ~keyIndex
+        val keepIndex = if (keyIndex == 0) 1 else 0
```

Comment:
I can't imagine what I was trying to achieve here or how I overlooked this 😆 All we are trying to do is select the other index in a 2-element collision node

Reply:
`1 - keyIndex` would also seem to be correct. As `keyIndex` can only be `0` or `1`, it would give the same result as the if/else. I can change it, if that seems better.

Reply:
Yeah. I expect you to school me about now, but I would think this could be more amenable to optimization than a conditional lol 😜